Converted JSON files into smaller CSV files

bsilver3 opened the pull request:
I went ahead and converted the original JSON files into CSV files that you guys can download in order to save space on GitHub and avoid using the JSON files to fix GitHub repo pushing issues.
I'm going to start clustering next, what do you think I should start with?

bsilver3 committed:
Imported first split

Temporary local file for now, looked at structure in a dataframe

bsilver3 committed:
Organization and Exported CSV files from original JSON files

sterlingthakidd reviewed (commented):
everything looks good

bsilver3 committed:
Started main analysis file

Imported CSV files and imported packages, going to start encoding and scaling data to begin next class

bsilver3 committed:
Wrangled and scaled data, attempted to incorporate some algorithms

Will work on finding the best way to encode categorical data later (not using one-hot encoding, since it adds too many features because of the amount of unique values, or label encoding, because it makes relationships that didn't exist before)

bsilver3 committed:
Update main.ipynb

bsilver3 committed:
Implemented model fully, and began preprocessing for predictions
